python/pysmurf/client/command/smurf_command.py

```diff
@@ -3873,6 +3873,7 @@ def set_rtm_slow_dac_enable(self, dac, val=2, **kwargs):
 
         # only ever set this to 0x2 or 0xE (enable or disable)
         if (val != 0x2) and (val != 0xE):
+            self.log("RTM dac val must be 0x2 or 0xE. Setting to 0x2.")
```

Comment:
Confusing, let's make it say something like "RTM dac enable val" since otherwise user might think it means RTM DAC data value.

Reply:
Looking at this more, it's odd that the pysmurf function is called `set_rtm_slow_dac_enable` for this register since it's not exactly an enable.  John gives the register the name DacCtrlReg (see https://confluence.slac.stanford.edu/display/SMuRF/RTM+firmware?preview=/232089354/233309457/Cryo_RTM_setup_note_v2.pdf) and says that register "configures AD5790 analog output."  I think in the DAC datasheet this register is called the "Software Control Register."  We should change pysmurf to match.  E.g. `set_rtm_slow_dac_control` instead of `set_rtm_slow_dac_enable`.

Reply:
Created new issue https://github.com/slaclab/pysmurf/issues/568